tests/packagedcode/test_chef.py

```diff
@@ -28,11 +28,15 @@
 
 from collections import OrderedDict
 import os.path
+from unittest.case import skipIf
```

Comment:
Use `pytest.xmark.skipif` instead as `skipIf` did not mesh well in the past with the other use of `pytestmark = pytest.mark.scanpy3`